http3/conn.go

```diff
@@ -112,8 +114,37 @@ func (c *connection) openRequestStream(
 	c.streams[str.StreamID()] = datagrams
 	c.streamMx.Unlock()
 	qstr := newStateTrackingStream(str, c, datagrams)
-	hstr := newStream(qstr, c, datagrams)
-	return newRequestStream(hstr, requestWriter, reqDone, c.decoder, disableCompression, maxHeaderBytes), nil
+	var rsp = new(http.Response)
```

Comment:
To make `gofumpt` happy:
```suggestion
	rsp := &http.Response{}
```